### PR TITLE
added process graphs for use case 1 with JRC backend

### PR DESCRIPTION
**Pull request by kempenep**

### CODE REVIEW

File: TUW_Radar_Image_Compositing/JSON/jrc_uc1_temp.json [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+      },
+      "description": "Merge RG and B data cubes to one with all three images in the band labels R, G and B"
+    },
+    "saveresult1": {
+      "process_id": "save_result",
+      "arguments": {
+        "data": {
+          "from_node": "mergecubes2"
+        },
+        "format": "GTIFF-THUMB",
```
Comment (m-mohr) [inline TUW_Radar_Image_Compositing/JSON/jrc_uc1_temp.json:200]: Are you sure this is really GTIFF-THUMB? That is a GEE only thing usually...

Comment (kempenep) [reply TUW_Radar_Image_Compositing/JSON/jrc_uc1_temp.json]: That is an error indeed. Should be GTIFF instead

**Comment by m-mohr** [comment]
Something is wrong with this PR, so many commits and changes in unrelated files, also has merge conflicts. I can't merge this as it is... @kempenep

**Comment by kempenep** [comment]
I suggest I create a new branch from the current master and request a pull from there

**Comment by m-mohr** [comment]
Yes, that sounds like a good idea.

**Pull request closed without merging**